src/canister_tests/src/framework.rs

```diff
@@ -32,7 +32,21 @@ lazy_static! {
         get_wasm_path("II_WASM".to_string(), &def_path).expect(&err)
     };
 
-    /** The Wasm module for the _previous_ build, or latest release, which is used when testing
+    /** The Wasm module for the current archive build, i.e. the one we're testing */
+    pub static ref ARCHIVE_WASM: Vec<u8> = {
+        let def_path = path::PathBuf::from("..").join("..").join("archive.wasm");
+        let err = format!("
+        Could not find Archive Wasm module for current build.
+
+        I will look for it at {:?}, and you can specify another path with the environment variable ARCHIVE_WASM (note that I run from {:?}).
+
+        In order to build the Wasm module, please run the following command:
+            ./scripts/build --archive
```

Comment:
Should we release the canister? Or alternatively, should we suggest a command to run everything but the archive canister tests?